src/main/java/uk/gov/hmcts/reform/sscs/service/AppInsightsService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.applicationinsights.telemetry.EventTelemetry;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.model.Message;
+
+@Slf4j
+@Service
+public final class AppInsightsService {
+
+    private static final TelemetryClient CLIENT = new TelemetryClient();
+
+    private AppInsightsService() {
+        // Gradle style check
+    }
+
+    public static void sendAppInsightsEvent(Message message) throws JsonProcessingException {
+        String serialisedMessage = messageToJson(message);
+
+        CLIENT.trackEvent(new EventTelemetry(serialisedMessage));
+        log.info("Event {} sent to AppInsights for Case ID {}", message, message.getCaseID());
+    }
+
+    private static String messageToJson(Message message) throws JsonProcessingException {
```

Comment:
Sorry but I think we should add a couple of tests for this. I know JPEs shouldn't happen normally but I'm worried about a library bump causing issues silently and us losing these critical events.